src/main/resources/assets/eln/lang/en_US.lang

```diff
@@ -261,7 +261,7 @@ Street_Light.name=Streetlight
 Suspended_Lamp_Socket.name=Suspended Lamp Socket
 Tachometer.name=Tachometer
 Temperature_Probe.name=Temperature Probe
-Thermal_Probe.name=Thermal Probe
+Thermal_Probe.name=Inline Temperature Probe
```

Comment:
IIRC this can also measure thermal wattage. If so, then it isn't merely a temperature probe. If not... that would be a useful addition.

Reply:
It does measure the wattage. Perhaps they should be called ```Thermal Probe``` and ```Inline Thermal Probe```? I'm not really sure. We could also make it more clear in the tip strip that it does thermal watts (which I think it may already do)

Reply:
If we are renaming those things, why not rename all of them to ... Sensor? IMHO this word is much more popular than "Probe".

Reply:
For the icons I propose another change:
How about to change the inline probe (or sensor) icon so that it is clear that the probe is inline? Like this:
![electricalprobe](https://user-images.githubusercontent.com/6661117/34563580-6c8b31bc-f153-11e7-87ce-394fb2e8f072.png)

Reply:
Oh, I really like the look of that. Indeed, sensor does sound much better. I will work on that real soon.

Some people are mentioning that we should consolidate the probes to one of each type. I think that for now we push this, and then later I can try consolidating the sensors together - combining them is (for me) significantly harder to do.

Reply:
The two types of sensors makes imho a lot of sense:
For the inline version, the connection must be opened in order to place it and a cable has to be placed into the sensor, the other sensor just connects to the side of the cable with the drawback that serial physical values as electrical current (and electrical power, as current is needed to calculate the power) and thermal power can not be measured with that kind of probe.

I do not see a simple way to combining them into one type for each...

Can you revert the icons in your PR so that I can push new versions without conflicts to dev?

Reply:
I would say push your icons now, and I will pull out my new icons. I would mention however that I don't think this is finished yet - I still need to fix the translations so that the text is properly translated, from what I saw, it was not doing it right. Perhaps I'm missing some part of the build process - I was told that there's something that does it automatically.

I really dislike the ```sensor``` icons (the non-inline ones), they feel really busy and look like batteries or capacitors. One of them was labeled ```c```, which made even less sense. There's no industry standard icons for that (as far as I'm aware), so it's up to our imagination to put something good there.